composer/models/huggingface.py

```diff
@@ -139,6 +139,102 @@ def __init__(self,
 
         self.dummy_forward_called = False
 
+    @staticmethod
+    def write_huggingface_tokenizer_from_composer_checkpoint(
```

Comment:
lets rename this function to `load_huggingface_tokenizer_from_saved_state` to more accurately describe what is does. Also, please add a docstring briefly describing what the function does, and the arguments (you can follow the format of other docstrings in this file)